CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 0.49.0
+
+- A new configuration option `downstream_branch_name` has been added,
+  which is meant to be used in source-git projects and allow users
+  to customize the name of the branch in dist-git which corresponds
+  to the current source-git branch. (#1555)
+- We have reverted functionality of Packit that allowed you to have set only
+  specific targets for Copr repositories. This functionality was introduced
+  a while ago and was found to be not very beneficial in case of differently
+  configured Copr jobs building in parallel in the same Copr repository. (#1551)
```

Comment:
I am not sure if this one is relevant to the CLI users.

Reply:
ok, removed, thanks